### PR TITLE
Add initial support for Polyhedra

### PR DESCRIPTION
**Pull request by blegat**
Implements the Polyhedra.jl interface so that QHull can be used through this common API.

### CODE REVIEW

**Comment by blegat** [comment]
@davidavdav On Travis it gets the error
```
ERROR: LoadError: KeyError: key "ConvexHull" not found
 in getindex(::PyCall.PyObject, ::String) at /home/travis/.julia/v0.5/PyCall/src/PyCall.jl:261
 in getindex(::PyCall.PyObject, ::Symbol) at /home/travis/.julia/v0.5/PyCall/src/PyCall.jl:266
 in chull(::Array{Float64,2}) at /home/travis/.julia/v0.5/QHull/src/QHull.jl:33
 in include_from_node1(::String) at ./loading.jl:488
 in process_options(::Base.JLOptions) at ./client.jl:262
 in _start() at ./client.jl:318
while loading /home/travis/.julia/v0.5/QHull/test/runtests.jl, in expression starting on line 12
```
Do you know what causes this ?

**Comment by davidavdav** [comment]
> Do you know what causes this ?

I have no idea... Could the python ConvexHull interface not be installed on Travis?

File: src/polyhedron.jl [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    V = ch.points[ch.vertices, :]
+    vnored = SimpleVRepresentation(V)
+    A = ch.facets[:, 1:N]
+    b = ch.facets[:, N+1]
+    h = SimpleHRepresentation(A, b)
+    vnored, h
+end
+
+function qhull{N}(h::SimpleHRepresentation{N, Float64})
+    A = h.A
```
Comment (blegat) [inline src/polyhedron.jl]: Good point, that's corrected

**Comment by davidavdav** [comment]
Are we sure that `ConvexHull` is in debian package `python-scipy`?

**Comment by blegat** [comment]
What I don't get is that normally it should be installed by the `build.jl` script so why do we also need to install it with `sudo apt-get install` ?

**Comment by blegat** [comment]
Looking at Travis, we are installing `python-scipy (0.9.0+dfsg1-1ubuntu2)` while according to [this](http://stackoverflow.com/questions/19826271/error-using-convexhull-from-scipy) we need at least 0.12.0

**Comment by blegat** [comment]
I found a way to make travis work :) If you set `ENV["PYTHON"]=""` then `PyCall.jl` will use the python version of `Conda.jl` and the python packages you need will be installed automatically when you need them :)

**Pull request merged**